wrims/build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
     implementation fileTree(dir: 'libs/x64', include: ['*.jar'])
```

Comment:
Why are there any dependencies left in this subproject? Shouldn't they now be provided by third-party?

Reply:
This project has been updated to use the third-party module.